doc/tor.1.txt

```diff
@@ -3369,260 +3319,260 @@ Tor catches the following signals:
+**!invalid**, then the descriptors are accepted but marked in the directory as
+not valid, that is, not recommended.
+
+|`**__DataDirectory__/v3-status-votes**`
+|Only for v3 authoritative directory servers. This file contains
+status votes from all the authoritative directory servers.
+
+|`**__CacheDirectory__/unverified-consensus**`
+|Contains a network consensus document that has been downloaded,
+but which we didn't have the right certificates to check yet
+
+|`**__CacheDirectory__/unverified-microdesc-consensus**`
+|Contains a microdescriptor-flavored network consensus document
+that has been downloaded, but which we didn't have the right certificates
+to check yet.
+
+|`**__DataDirectory__/unparseable-desc**`
+| Onion server descriptors that Tor was unable to parse are dumped to this
+  file. Only used for debugging.
+
+|`**__DataDirectory__/router-stability**`
+|  Only used by authoritative directory servers. Tracks measurements for
+  router mean-time-between-failures so that authorities have a fair idea of
+  how to set their Stable flags.
+
+|`**__DataDirectory__/stats/dirreq-stats**`
+| Only used by directory caches and authorities. This file is used to
+  collect directory request statistics.
+
+|`**__DataDirectory__/stats/entry-stats**`
+|Only used by servers. This file is used to collect incoming connection
+statistics by Tor entry nodes.
+
+|`**__DataDirectory__/stats/bridge-stats**`
+|Only used by servers. This file is used to collect incoming connection
+statistics by Tor bridges.
+
+|`**__DataDirectory__/stats/exit-stats**`
+| Only used by servers. This file is used to collect outgoing connection
+  statistics by Tor exit routers.
+
+|`**__DataDirectory__/stats/buffer-stats**`
+|Only used by servers. This file is used to collect buffer usage
+history.
+
+|`**__DataDirectory__/stats/conn-stats**`
+|Only used by servers. This file is used to collect approximate connection
+history (number of active connections over time).
+
+|`**__DataDirectory__/stats/hidserv-stats**`
+|Only used by servers. This file is used to collect approximate counts
+of what fraction of the traffic is hidden service rendezvous traffic, and
+approximately how many hidden services the relay has seen.
+
+|`**__DataDirectory__/networkstatus-bridges**`
+|Only used by authoritative bridge directories. Contains information
+about bridges that have self-reported themselves to the bridge
+authority.
+
+|`**__DataDirectory__/approved-routers**`
+|_Authorities only_. This file is used to configure which relays are
+known to be valid, invalid, and so forth.
+
+|`**__HiddenServiceDirectory__/hostname**`
+| The <base32-encoded-fingerprint>.onion domain name for this hidden service.
+  If the hidden service is restricted to authorized clients only, this file
+  also contains authorization data for all clients. +
+  NOTE: The clients will ignore any extra subdomains prepended to a hidden
+  service hostname. Suppose you have "xyz.onion" as your hostname, you
```

Comment:
This is a comma splice; I think you could fix it by changing "Suppose" to "Supposing".